### PR TITLE
Upgrade aliyun api to retry timeout

### PR DESCRIPTION
**Pull request by chunyong-lin**
to: @ryandeivert @Ryxias 
cc: @airbnb/streamalert-maintainers
related to:
resolves:  #926

## Background
In aliyun-python-sdk-core v2.8.5 SDK doesn't handle retries for timeout. This causes Aliyun app to crash more frequently recently. In v2.13.5, it supports retries for timeout in [this PR](https://github.com/aliyun/aliyun-openapi-python-sdk/pull/163/files).

~Also, this PR fix a duplicated dependency `policyuniverse` causes `rules_engine` deployment to be failed when it builds dependencies for `rules_engine`.~ This fix goes to #970 and merged to `master` branch while rest changes will go to `release-2-3-0` branch.

Upgrade some dependencies' versions.

## Changes

* Upgrade aliyun-python-sdk-core version to `2.13.5` and recompiled `aliyun-python-sdk-actiontrail==2.0.0_dependencies.zip` package for lambda.
* Upgrade some dependencies to resolve some security concerns
  * urllib3==1.24.2 [CVE-2019-11324](https://nvd.nist.gov/vuln/detail/CVE-2019-11324)
  * Jinja2==2.10.1 [CVE-2019-10906](https://nvd.nist.gov/vuln/detail/CVE-2019-10906)
  * pyyaml==5.1.2 [CVE-2017-18342](https://nvd.nist.gov/vuln/detail/CVE-2017-18342)
  * And other dependencies

## Testing
The change has been tested in staging environment
* Deployed a new StreamAlert
* Deployed an Aliyun App using testing account
* Generate Aliyun ActionTrail events to trigger classifier, rules_engine and alert processor. And the pipeline works correctly.

### CODE REVIEW

**Comment by ryandeivert** [comment]
@Ryxias `#nosec` prevents `bandit` from complaining that something is 'insecure'

**Comment by chunyong-lin** [comment]
Yes. I upgrade bandit to new version and it alerts on more things. Most of them are in `tests/` directory, so it is okay to disable bandit on testing code.

File: stream_alert/apps/_apps/salesforce.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ class SalesforceApp(AppIntegration):
     ReportExport:
         events contain details about reports that a user exported.
     """
-    _SALESFORCE_TOKEN_URL = 'https://login.salesforce.com/services/oauth2/token'
+    _SALESFORCE_TOKEN_URL = 'https://login.salesforce.com/services/oauth2/token' #nosec
```
Comment (ryandeivert) [inline stream_alert/apps/_apps/salesforce.py]: definitely a nitpick, but to be consistent with else where, can you update these comments to be `# nosec` (space after `#`)

Comment (chunyong-lin) [reply stream_alert/apps/_apps/salesforce.py]: 😨 ooops, I thought had to be one word. LOL. Will correct it after rebase this branch.

**Pull request merged**